xoinvader/tests/test_enemy_wave.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+
+from xoinvader.enemy_wave import EnemyWave
+
+
+def test_wave():
+    e = EnemyWave()
+
+    a = 0
+    b = 0
+    def inc_a():
```

Comment:
You will change GameObject attributes not some variables at the middle of nowhere. Rewrite it to simple class please.